[2024-06-22T12:08:24.703+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_ml_pipeline_class_reg.run_modeling_classification manual__2024-06-22T12:08:18.775039+00:00 [queued]>
[2024-06-22T12:08:24.717+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_ml_pipeline_class_reg.run_modeling_classification manual__2024-06-22T12:08:18.775039+00:00 [queued]>
[2024-06-22T12:08:24.718+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-06-22T12:08:24.736+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): run_modeling_classification> on 2024-06-22 12:08:18.775039+00:00
[2024-06-22T12:08:24.744+0000] {standard_task_runner.py:60} INFO - Started process 2407 to run task
[2024-06-22T12:08:24.747+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'DAG_ml_pipeline_class_reg', 'run_modeling_classification', 'manual__2024-06-22T12:08:18.775039+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/run_data_wrangling_class_reg.py', '--cfg-path', '/tmp/tmp0x1ct8sp']
[2024-06-22T12:08:24.750+0000] {standard_task_runner.py:88} INFO - Job 29: Subtask run_modeling_classification
[2024-06-22T12:08:24.815+0000] {task_command.py:423} INFO - Running <TaskInstance: DAG_ml_pipeline_class_reg.run_modeling_classification manual__2024-06-22T12:08:18.775039+00:00 [running]> on host 0fdcf3d7bc80
[2024-06-22T12:08:24.897+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_ml_pipeline_class_reg' AIRFLOW_CTX_TASK_ID='run_modeling_classification' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T12:08:18.775039+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-22T12:08:18.775039+00:00'
[2024-06-22T12:08:24.899+0000] {logging_mixin.py:188} INFO - Looking for .env file at: /opt/***/config/.env
[2024-06-22T12:08:24.925+0000] {credentials.py:1278} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2024-06-22T12:08:25.957+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/sklearn/base.py:348: InconsistentVersionWarning: Trying to unpickle estimator LogisticRegression from version 1.4.2 when using version 1.3.2. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:
https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations
  warnings.warn(

[2024-06-22T12:08:26.184+0000] {logging_mixin.py:188} INFO - Demurrage Prediction file saved to s3://logimo-outbound/ship_cost_predictor/ship_cost_classification/Data_Pred_Demurrage_True_Join.csv for regressor ingestion
[2024-06-22T12:08:26.187+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-06-22T12:08:26.199+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=DAG_ml_pipeline_class_reg, task_id=run_modeling_classification, execution_date=20240622T120818, start_date=20240622T120824, end_date=20240622T120826
[2024-06-22T12:08:26.249+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-06-22T12:08:26.271+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
